src/main/java/frc/robot/RobotContainer.java

```diff
@@ -145,7 +145,7 @@ private void configureButtonBindings() {
         // Enable Shooter Magazine Combo While Operator A Button Held/
 
         new AxisButton(operator, XboxController.Axis.kRightTrigger.value)
-            .whileHeld(new ParallelCommandGroup(new ShooterRPM(this.shooter, 4300 / 60),
+            .whileHeld(new ParallelCommandGroup(new ShooterRPM(this.shooter, 5600 / 60),
```

Comment:
Need to reset this to whatever it needs to be at the tape line. I think 4000 based off the testing.

Reply:
fixed